kafka/integration_test.go

```diff
@@ -233,7 +233,7 @@ func producerTest(t *testing.T, testname string, testmsgs []*testmsgType, pc pro
 
 	if testmsgs == nil {
 		createTestMessages()
-		testmsgs = pAllTestMsgs
+		testmsgs = p0TestMsgs
```

Comment:
Why this change?

Reply:
I'll rebase this out - I had thought that having the invalid partition message in pAllTestMsgs would cause problems but it seems like it all works.